### ISSUE TITLE
An activity feed line for when offers are cancelled automatically.

### ISSUE THREAD

**Issue by bartervg** [bartervg/barter.vg#227]
With the change, I wasn't able to reproduce this expiration attack. This offer https://barter.vg/u/a0/o/3912363/ cancelled harmlessly without setting luckz to unavailable. It may be useful to have an activity feed line for when offers are cancelled automatically.

There isn't an activity feed item when an offer is cancelled. Either automatically or by the offer creator.

_Originally posted by @bartervg in https://github.com/bartervg/barter.vg/issues/213#issuecomment-770909973_

**Comment by Luckz** [comment]
> There isn't an activity feed item when an offer is cancelled. Either automatically or by the offer creator.

Having abandoned offers auto-cancel and usually never found again by anyone is _a feature_ though, is it not?
Beyond the rare cases when you don't look at an offer for some hours and suddenly it's gone, surely an intrepid offer creator should be able to change their mind in mid-creation without leaving a too obvious lasting trace.

**Comment by bartervg** [comment]
> Having abandoned offers auto-cancel and usually never found again by anyone is a feature though, is it not?

Yes, I think this is the typical use case. However, in the test, I wanted to know when it was cancelled. All that's recorded is `18 hours ago ✐ Barter.vg edited offer`. The cancel activity doesn't need to appear in the main offer feed, but could be limited to the specific offer: `15 hours ago offer was automatically cancelled`

**Comment by bartervg** [comment]
Bug: rare types of cancelled offers do not show any status 
![image](https://user-images.githubusercontent.com/28576132/106523578-53a04980-64af-11eb-920e-8f39b797cb90.png)
